### ISSUE TITLE
IOS: App crashes after calling ImagePicker.openPicker

### ISSUE THREAD

**Issue by qnarik-avetisyan** [ivpusic/react-native-image-crop-picker#1817]
App crashes after calling ImagePicker.openPicker on IOS with error - Could not find a storyboard named 'QBImagePicker' in bundle NSBundle. (MAC OS M1, Xcode 13)
I tried to add QBImagePicker.storyboard to Copy Bundle Resources, but getting same error again.
Picker is opening and working as expected on android.

**Comment by Allenkardic** [comment]
i am having same issues on ios is there any solution yet

**Comment by cozmoc** [comment]
seems to be happening only on M1

**Comment by Nziranziza** [comment]
I am also experiencing the similar issue

**Comment by hiephoang190393** [comment]
Add this key: _NSPhotoLibraryUsageDescription_ to Info.plist. This issue will be resolved.

**Comment by FayyazAliKhan1** [comment]
I am also facing this while selecting random images just like some from start some from between & some from last images (e.g gallery) [1867](https://github.com/ivpusic/react-native-image-crop-picker/issues/1867)

**Comment by jacquesdev** [comment]
> Add this key: _NSPhotoLibraryUsageDescription_ to Info.plist. This issue will be resolved.

This is not the issue - we have this permission - and the crash is happening

**Comment by arwys** [comment]
Yes, and i ve got an issue , sometimes i can upload the images, but sometimes it couldn’t (invalid file), but in Android it works fine 100%

**Comment by jacquesdev** [comment]
@arwys - but sometimes it couldn’t (invalid file). Do you know what this errror is?

**Comment by arwys** [comment]
I dont know what is the error to be honest, i am on 0.64, its confusing me till now

**Comment by inkCrazy** [comment]
我同事选择视频会出现界面卡死或者闪退，一些视频好的，一些不行，他把不好的视频 发给我,我上传没问题。
Xcode 13.2.1 iPhone12  iOS15.4.1
"react-native-image-crop-picker": "^0.33.4",
"react-native": "0.63.5",

**Comment by yasinosman** [comment]
same here,

I am using `react-native-image-crop-picker: 0.38.0`

I have the relevant usage descriptions in my `Info.plist` file:
```
<key>NSPhotoLibraryUsageDescription</key>
<string>Please allow us to access to your photo library to upload photos.</string>
<key>NSCameraUsageDescription</key>
<string>Please allow us to access to your camera to take photos.</string>
```

When I try to call `ImagePicker.openPicker`, I receive the following error:
```
"Could not find a storyboard named 'QBImagePicker' in bundle NSBundle
```
